tests/scripts/thread-cert/Cert_5_3_01_LinkLocal.py

```diff
@@ -75,6 +99,105 @@ def test(self):
         # 8
         self.assertTrue(self.nodes[LEADER].ping(config.LINK_LOCAL_All_THREAD_NODES_MULTICAST_ADDRESS))
 
+    def verify(self, pv):
+        pkts = pv.pkts
+        pv.summary.show()
+
+        LEADER = pv.vars['LEADER']
+        LEADER_LLA = pv.vars['LEADER_LLA']
+        ROUTER_LLA = pv.vars['ROUTER_LLA']
+
+        # Step 1: Build the topology as described
+        pv.verify_attached('ROUTER')
+
+        # Step 2: Leader Send a Fragmented ICMPv6 Echo Request to DUT’s
+        #         MAC extended address based Link-Local address
+        #         The DUT MUST respond with an ICMPv6 Echo Reply
+
+        _pkt = pkts.filter_ping_request().\
+            filter_ipv6_src_dst(LEADER_LLA, ROUTER_LLA).\
+            filter(lambda p: p.icmpv6.data.len == 256).\
```

Comment:
avoid magic number `256` in this test case

Reply:
add a const DATA_LEN = 256